internal/synclet/synclet.proto

```diff
@@ -30,6 +34,24 @@ message UpdateContainerRequest {
     bytes tar_archive = 2;
     repeated string files_to_delete = 3;
     repeated Cmd commands = 4;
+    Context context = 5;
+}
+
+enum LogLevel {
+    INFO = 0;
+    VERBOSE = 1;
+    DEBUG = 2;
+}
+
+message LogMessage {
+    LogLevel level = 1;
+    bytes message = 2;
+}
+
+message UpdateContainerReply {
+    LogMessage logMessage = 1;
```

Comment:
protobuf style is slug_case for field names
https://developers.google.com/protocol-buffers/docs/style